packages/tradershub/src/features/cfd/screens/Jurisdiction/JurisdictionScreen.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { Dispatch, useEffect, useMemo } from 'react';
+import { useAvailableMT5Accounts, useMT5AccountsList } from '@deriv/api';
+import { Provider } from '@deriv/library';
+import { qtMerge, Text } from '@deriv/quill-design';
+import { THooks } from '../../../../types';
+import { useDynamicLeverageModalState } from '../../components/DynamicLeverageContext';
+import { Jurisdiction } from '../../constants';
+import { JurisdictionCard } from './JurisdictionCard';
+import { JurisdictionTncSection } from './JurisdictionTncSection';
+
+type TJurisdictionScreenProps = {
+    isCheckBoxChecked: boolean;
+    selectedJurisdiction: THooks.AvailableMT5Accounts['shortcode'];
+    setIsCheckBoxChecked: Dispatch<React.SetStateAction<boolean>>;
+    setSelectedJurisdiction: Dispatch<React.SetStateAction<string>>;
+};
+
+const JurisdictionScreen = ({
+    isCheckBoxChecked,
+    selectedJurisdiction,
+    setIsCheckBoxChecked,
+    setSelectedJurisdiction,
+}: TJurisdictionScreenProps) => {
+    const { getCFDState } = Provider.useCFDContext();
+    const { data, isLoading } = useAvailableMT5Accounts();
```

Comment:
data seems vague please use a name so we can infer what data it contains
```suggestion
    const { data: availableMT5Accounts, isLoading } = useAvailableMT5Accounts();
```

Reply:
Updated.